src/backend/tools/index.ts

```diff
@@ -117,24 +117,29 @@ async function installOrUpdateTool(tool: Tool) {
 
   const extractDestination = join(toolsPath, tool.name, latestVersion)
   mkdirSync(extractDestination, { recursive: true })
-  try {
-    await extractFiles({
-      path: latestVersionArchivePath,
-      destination: extractDestination,
-      strip: tool.strip ?? 1
+  const extractCommand = `${tool.extractCommand} ${latestVersion} -C ${toolsPath}/${tool.name}`
+  await execAsync(extractCommand)
```

Comment:
this begs for issues with space in the path. It may be better to use spawn here, or if we really need exec, let's at least quote variables that may contain spaces

Reply:
changed